src/views/virtualmachines/details/tabs/configuration/details/components/DetailsSectionBoot.tsx

```diff
@@ -49,7 +47,7 @@ const DetailsSectionBoot: FC<DetailsSectionBootProps> = ({
 
   useEffect(() => {
     expandURLHash(getSearchItemsIds(getDetailsTabBootIds(vm)), location?.hash, setIsExpanded);
-  }, [vm, location?.hash]);
+  }, [vm, location?.hash, setIsExpanded]);
```

Comment:
no need to include setState into deps - do u have extensions that announce if some deps are missing?

Reply:
The method comes from our custom hook `useToggle()`  - if it's not added to the list I get warnings in IDE